front/src/SRRButton.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+/**
+ * Create a standard RSS Button
```

Comment:
Why is this entire component namespaced? Isn't it usable as a general button as well?

Also, RSS -> SRR